client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -81,34 +119,71 @@ function ItemDetail({
       >
         <CloseIcon />
       </IconButton>
-      <div style={{ width: "100vw", height: "60vh", overflow: "hidden" }}>
-        <img
-          src={item.media ? item.media : PLACEHOLDER_ITEM_MEDIA}
-          style={{ height: "100%", width: "100%", objectFit: "cover" }}
-        />
-      </div>
+      <div
+        style={{
+          width: "100vw",
+          height: `${Math.floor((1 - footerPercentageHeight) * 100)}vh`,
+          overflow: "hidden",
+          backgroundPosition: "center",
+          backgroundSize: "cover",
+          backgroundImage: `url('${
+            item.media ? item.media : PLACEHOLDER_ITEM_MEDIA
+          }')`,
+        }}
+      />
       <div style={{ width: "100vw" }}>
         {/*TODO(#198): zIndex layering helper class needed to manage layers on page*/}
         <div
           style={{
             position: "absolute",
             zIndex: 1003,
             width: "100%",
-            height: `${cornerRoundingPercentage}vw`,
-            marginTop: `-${cornerRoundingPercentage}vw`,
-            borderTopLeftRadius: `${cornerRoundingPercentage}vw`,
-            borderTopRightRadius: `${cornerRoundingPercentage}vw`,
+            height: `${cornerRoundingRadius}`,
+            marginTop: `-${cornerRoundingRadius}`,
+            borderTopLeftRadius: `${cornerRoundingRadius}`,
+            borderTopRightRadius: `${cornerRoundingRadius}`,
             backgroundColor: "#FFFFFF",
+            textAlign: "center",
           }}
-        />
+        >
+          {/*Floating Expand button*/}
```

Comment:
nit: remove?

Reply:
see below, we need to surround comments with the brace for jsx

Reply:
ah okay! then its fine